assets/css/custom.css

```diff
@@ -70,12 +70,13 @@ body {
   justify-content: center;
   flex-direction: row;
   margin-top: 1%;
+  overflow: scroll;
```

Comment:
this shouldn't be necessary...should happen automatically. if for some reason it's not, you can use `overflow: auto`. `overflow: scroll` will put a scrollbar there whether it's needed or not